packages/engine-server/src/enginev2.ts

```diff
@@ -469,9 +469,6 @@ export class DendronEngineV2 implements DEngine {
     // If provided, we render the given note entirely. Otherwise find the note in workspace.
     if (!note) {
       note = this.notes[id];
-    } else {
```

Comment:
will this be a problem if a note has a note reference to itself?

Reply:
It _shouldn't_ be - the note gets included as a part of the dependency set for unified processing - but @Harshita-mindfire could you do a quick test to make sure that's the case?

Reply:
Yep, works fine.
![image](https://user-images.githubusercontent.com/84971366/198522859-067f9fbf-3bc5-477d-b16c-a4c616503580.png)